release/_releases_json.sh

```diff
@@ -1,6 +1,21 @@
 #!/bin/bash
 
+function _add_new_version {
+	echo "Add ${LIFERAY_RELEASE_VERSION} version to supported-${LIFERAY_RELEASE_PRODUCT_NAME}-versions.txt"
+
+	if [ -n "${LIFERAY_RELEASE_PRODUCT_NAME}" ]
+	then
+		echo -en "\n${LIFERAY_RELEASE_VERSION}" >> "${_RELEASE_ROOT_DIR}"/supported-"${LIFERAY_RELEASE_PRODUCT_NAME}"-versions.txt
+	else
+		echo "    LIFERAY_RELEASE_PRODUCT_NAME: Set to \"portal\" or \"dxp\" "
+		echo "    LIFERAY_RELEASE_VERSION=Set a version to add it in supported-{product}-versions.txt"
+		echo "Example: LIFERAY_RELEASE_PRODUCT_NAME=dxp LIFERAY_RELEASE_VERSION=2024.q1 ${0}"
```

Comment:
@Wykenz, shouldn't we adjust the tabulation here as in https://github.com/liferay/liferay-docker/blob/a7c23c11829c2802d5cb3d96b90eb1a44b803804/release/build_release.sh#L198-L215?